Bumerang/app/src/main/res/values/strings.xml

```diff
@@ -45,10 +45,15 @@
     <string name="updated_profile">Your profile has been successfully updated!</string>
     <string name="facebook_first_name">First Name</string>
 
+    <string name="unable_to_display_request_detail">Sorry, we weren\'t able to get the details of that request. Please try again!</string>
+    <string name="unable_to_accept_own_request">Nope! You can\'t accept your own request.</string>
```

Comment:
Can this be Sorry, but you can't accept your own request. 

Just my opinion though.